### PR TITLE
CI/CD: update sgx-tools testcases by add the quote type

### PR DESCRIPTION
**Pull request by hustliyilin**
Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

**Comment by hustliyilin** [comment]
FYI: @jiazhang0 @haosanzi : 
- Please not merge this pr before approving [sgx-tools: support to get target info and quote both in epid and ecdsa modes ](https://github.com/alibaba/inclavare-containers/pull/584).
- approve #584 and merge this pr and add the test-before-checking label in for #584 , and merge #584 finally.

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**
